lib/ansiblelint/formatters/__init__.py

```diff
@@ -1,20 +1,40 @@
 """Output formatters."""
+import os
+
 try:
     from ansible import color
 except ImportError:
     from ansible.utils import color
-from ansiblelint.utils import normpath
 
 
-class Formatter(object):
+class BaseFormatter(object):
+    """Formats ansible lint output.
```

Comment:
Class docstring shouldn't start with a verb. Classes don't do actions, unlike methods.
```suggestion
    """Formatter of ansible-lint output.
```